config/jobs/kubernetes/sig-node/dra.jinja

```diff
@@ -0,0 +1,120 @@
+{%- if header %}{%- if kind == "ci" %}periodics:{%- else %}presubmits:
```

Comment:
Can this `{%- if header %}` be dropped?

Reply:
Can you suggest how to do it? I don't like this trick either, but couldn't find a better way to make `periodics:` and `presubmits:` appear in a result content only once, at the beginning of the file.

Reply:
So that is the purpose - that wasn't clear to me. Can you perhaps add a comment to the `dra.jinja` explaining this trick?

I don't have a better alternative.